test/xbgpu/test_engine.py

```diff
@@ -1289,42 +1379,65 @@ def heap_factory(batch_index: int, present: np.ndarray) -> list[spead2.send.Heap
             assert stream_diff.diff("output_x_clipped_visibilities_total") == n_vis
             assert xbengine.sensors[f"{corrprod_output.name}.xeng-clip-cnt"].value == n_vis
 
-    def _patch_get_in_item(
+    def _patch_method(
         self,
         monkeypatch: pytest.MonkeyPatch,
+        method_name: str,
+        class_type: type[_T],
```

Comment:
Calls would read a little easier if these parameters were swapped, so that a patch to Class.method takes Class, method.

Reply:
Of course, you are too right. I was thinking about reordering as well, not sure why I left it in this "unintuitive" way.